protocols/luci-proto-ipv6/htdocs/luci-static/resources/protocol/map.js

```diff
@@ -86,5 +86,12 @@ return network.registerProtocol('map', {
 		o.datatype    = 'max(9200)';
 
 		o = s.taboption('advanced', form.Flag, 'legacymap', _('Use legacy MAP'), _('Use legacy MAP interface identifier format (draft-ietf-softwire-map-00) instead of RFC7597'));
+
+		o = s.taboption('advanced', form.Flag, 'isp_fix', _('Enable SNAT fix'), _('Apply SNAT fixes with certain ISPs'));
```

Comment:
Perhaps just 
```js
		o = s.taboption('advanced', form.Flag, 'isp_fix', _('Enable SNAT fix'), _('Only for certain ISPs'));
```

Then you avoid the weird variable name in the openwrt script portion also.

Reply:
Replaced with `snat_fix`.